src/components/Layout.js

```diff
@@ -1,6 +1,5 @@
 import React from "react";
-import Header from "./Header";
-import Footer from "./Footer";
+import { Footer, Header } from ".";
```

Comment:
`import { Header, Footer} from ".";`

Reply:
Good point. Maybe stick with alphabetical since the order in the DOM could change? What do you think?